media-types.md

```diff
@@ -3,6 +3,7 @@
 The following `mediaType` MIME types are used by the formats described here, and the resources they reference:
 
 - `application/vnd.oci.descriptor.v1+json`: [Content Descriptor](descriptor.md)
+- `application/vnd.oci.descriptor.foreign.v1+json`: [Content Descriptor](descriptor.md)
```

Comment:
How do you figure out what the real type is?  Docker [uses `application/vnd.docker.image.rootfs.foreign.diff.tar.gzip`][1], which is clearly a foreign version of their [`application/vnd.docker.image.rootfs.diff.tar.gzip`][2].

Stepping back, I [don't think we need this information for pulling][3].  However, “this content is not licensed for sharing” is a clear property that doesn't rely on vague ideas like “the usual channel”.  Maybe we can add a new descriptor.license field (an array of [SPDX identifiers][4]?), and refuse to push blobs which don't have such an array (and are therefore not shareable).

Blob license information would be nice (more in #71), but if we want to scope this more narrowly we could add a boolean descriptor.shareable to make life easier for publishers.

[1]: https://github.com/docker/distribution/pull/1725/files#diff-298c8e761320ff3dde0420467054f000R223
[2]: https://github.com/docker/distribution/pull/1725/files#diff-c9a5ec5e4eaff01148fd58b92fc0fd31R22
[3]: https://github.com/opencontainers/image-spec/pull/169#issuecomment-237364025
[4]: https://spdx.org/licenses/